system/settings/settings.xml

```diff
@@ -2192,12 +2192,6 @@
           <control type="spinner" format="string" />
         </setting>
         <setting id="videoscreen.limitedrange" type="boolean" label="36042" help="36359">
-          <requirement>
-            <or>
-              <condition>HAS_GL</condition>
-              <condition>HAS_DX</condition>
-            </or>
-          </requirement>
```

Comment:
this will make it visible on all platforms

Reply:
@FernetMenta  @lrusak something like this maybe: https://github.com/a1rwulf/xbmc/commit/3de3b71f524f622f90d6d15d4d7db059dce86c47